_dev/docker/ci/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# These builds occur in the CI. Reference .circleci/config.yml. They assume
```

Comment:
We now build three ci images instead of two. The introduction of the third image, made switching to a multistage build in single file feel much cleaner than referencing various image tags in different build files.